### PR TITLE
Empty results if no duplicates found

### PR DESCRIPTION
**Pull request by nikhilweee**
The `List duplicates` page currently displays all persons if no duplicates are found.
This fix resolves the issue.

### CODE REVIEW

**Comment by pbanaszkiewicz** [comment]
Please add a regression test in `test_finding_duplicates.py` to make sure that when no people match our criteria, we get an empty result set.

**Comment by nikhilweee** [comment]
I've added a test case to verify the result. I hope that this is what was required. Please review!

File: workshops/test/test_finding_duplicates.py [hunk 1]
```diff
@@ -3,6 +3,40 @@
...
+    def test_switched_names_persons(self):
+        rv = self.client.get(self.url)
+        switched = rv.context['switched_persons']
+        self.assertNotIn(self.harry, switched)
+        self.assertNotIn(self.kira, switched)
+        self.assertNotIn(self.batman, switched)
+        self.assertNotIn(self.ironman, switched)
+
+    def test_duplicate_persons(self):
+        rv = self.client.get(self.url)
```
Comment (pbanaszkiewicz) [inline workshops/test/test_finding_duplicates.py]: Can you please add docstrings to these tests explaining what is being tested and why? Thanks

**Comment by nikhilweee** [comment]
@pbanaszkiewicz - done updating docstrings!

**Comment by pbanaszkiewicz** [comment]
Excellent, thanks a lot!

**Pull request merged**